posts/2021-the-bottom-line.md

```diff
@@ -0,0 +1,177 @@
+---
+author: 'Mathlib community'
+category: 'overview'
+date: 2021-12-30 23:59:59+00:00
+description: ''
+has_math: true
+link: ''
+slug: 2021-the-bottom-line
+tags: ''
+title: 2021 - The Bottom Line
+type: text
+---
+
+Since the mathlib repository was created in summer 2017,
+each year has been bigger than the last. 
+As an end-of-year retrospective, we look at how the mathlib library and community have developed in 2021.
+
+# Statistics
+
+Since early 2019, mathlib has followed a "squash and merge" strategy for pull requests.
+One commit to the main branch corresponds to one merged pull request.
+
+| | Dec 31, 2020 | Dec 31, 2021 | Increase |
```

Comment:
This table formats badly in the GitHub preview but looks fine when built by Nikola.